app/controllers/facility_estimates_controller.rb

```diff
@@ -71,9 +71,28 @@ def create
     end
   end
 
+  def edit
+    set_collections_for_select
+  end
+
+  def update
+    expires_at = parse_usa_date(facility_estimate_params[:expires_at])
+
+    update_params = facility_estimate_params.merge(expires_at:)
```

Comment:
maybe we can move this over to the `facility_estimate_params` method since it's also used in the create action. What do you think?